Library/Homebrew/compat/pathname.rb

```diff
@@ -9,8 +10,12 @@ def cp(dst)
     dst
   end
 
+  # We won't change the name because of backward compatibility.
+  # So disable rubocop here.
+  # rubocop:disable Style/MethodName
```

Comment:
As before, please put it at the end of `def chmod_R(perms)` to limit the scope of this special comment.